centreon/www/front_src/src/Dashboards/components/DashboardLibrary/DashboardListing/atom.ts

```diff
@@ -14,8 +14,8 @@ export const sortFieldAtom = atom<string>('name');
 export const searchAtom = atom<string>('');
 
 export const viewModeAtom = atomWithStorage<ViewMode>(
-  'dashboards-view-mode',
-  ViewMode.List
+  'dashboards-view-mode-24.10',
```

Comment:
```suggestion
  'dashboards-view-mode',
```
No need to change the key here if the user already changed the view mode